helm-chart/templates/nextcloud-aio-talk-service.yaml

```diff
@@ -17,6 +18,9 @@ spec:
       port: {{ .Values.TALK_PORT }}
       protocol: UDP
       targetPort: {{ .Values.TALK_PORT }}
+    - name: "8081"
+      port: 8081
+      targetPort: 8081
```

Comment:
this port does not need to be published. How can I let it only work internally?

Reply:
The default type is `ClusterIP` and it will only be reachable from inside the cluster.

Reply:
okay good! Thanks!

Reply:
I suppose I might need to add another service file manuay that includes this port?

Reply:
Or can I limit the type somehow to specific ports?

Reply:
You need to split the internal ports to another service. There is no way to selectively publish ports from a single service.

Reply:
okay, I thought so. Thanks! :)

Reply:
will do in a follow-up.